tests/e2e/util/commonUtils.go

```diff
@@ -68,16 +68,27 @@ func WriteTempfile(tmpDir, prefix, suffix, contents string) (string, error) {
 
 // Shell run command on shell and get back output and error if get one
 func Shell(format string, args ...interface{}) (string, error) {
+	return sh(format, true, args)
+}
+
+// ShellMuteOutput run command on shell and get back output and error if get one
+// without logging the output
+func ShellMuteOutput(format string, args ...interface{}) (string, error) {
+	return sh(format, false, args)
+}
+
+func sh(format string, logOutput bool, args ...interface{}) (string, error) {
```

Comment:
From: [log](https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_istio/673/e2e-suite-rbac-no_auth/396/):  `kubectl create namespace [mixer-test-86d2f7b0834b489caeb6917b89]`

If you print `args` in `Shell(...)`, you get `[demo-test-cbd6eaaacfb5427ba65160b3999]`

and then you print `args` in `sh(...)`, you get `[[demo-test-cbd6eaaacfb5427ba65160b3999]]`

Wrapped twice.